bin/smoke

```diff
@@ -17,15 +17,14 @@ else
   tests="$*"
 fi
 
-yarn jsonify test/data.yml
-rm -rf results/
+./bin/unit_test "${tests}"
 
 echo "Smoking SDKs: ${tests} ..."
 
 if [[ $tests == *typescript* ]]
 then
   echo "Smoking Typescript SDK ..."
-  JEST_JUNIT_OUTPUT_DIR=results JEST_JUNIT_OUTPUT_NAME=smoketypescript.xml yarn jest packages/sdk --reporters=default --reporters=jest-junit
+  JEST_JUNIT_OUTPUT_DIR=results JEST_JUNIT_OUTPUT_NAME=ts_sdk_smoke.xml yarn jest packages/sdk/test --reporters=default --reporters=jest-junit
```

Comment:
curious why `packages/sdk/` has a separate `test/` dir? It seems like the convention is to have spec files alongside the source files? I can see the argument that `methods.spec.ts` doesn't unittest a particular file (in which case it might be better named something like `integration.spec.ts` or `smoke.spec.ts`) but still seems like the odd ball wrt to all the other `packages/` having their .spec.ts files side-by-side

Reply:
You're right that the TS convention is to have the test file next to the file being tested. 

I thought it best to violate that rule for tests that use our generated SDK files. I think the only thing that should be in the generated directories is the generated code. Because of the way jest tests, we need a distinct path, because `yarn jest packages/sdk` would result in running the tests for:

- packages/sdk
- packages/sdk-rtl
- packages/sdk-codegen
- packages/sdk-codegen-scripts

which is definitely not what we want.

We could rename the test source file, but it _is_ testing the methods, after all.